api/tests/test_views.py

```diff
@@ -1,64 +1,153 @@
 import pytest
+import json
 
+from django.test import TestCase
+from django.utils import timezone
+from rest_framework.test import RequestsClient
 
-def test_fire():
-    """
-    Employees related test
-    """
-    pass
+from ..factories import EmployeeFactory, IncomeFactory, FutureEventFactory, TodaysEventsFactory, PastEventFactory
 
+SERVER_ADDRESS = 'http://127.0.0.1:8000'
+CLIENT = RequestsClient()
+EMPLOYEES = 'employees'
+EVENTS = 'events'
+INCOMES = 'income'
 
-def test_hire():
-    """
-    Employees related test
-    """
-    pass
 
+def _url_not_detail(content, method='', options=''):
+    return SERVER_ADDRESS + '/api/{}/{}/{}'.format(content, method, options)
 
-def test_change_position():
-    """
-    Employees related test
-    """
-    pass
 
+def _url_detail(content, id, method='', options=''):
+    return SERVER_ADDRESS + '/api/{}/{}/{}/'.format(content, id, method, options)
 
-def test_summary_cost_of_salaries():
-    """
-    Employees related test
-    """
-    pass
 
+def _option(option, value):
+    return '?{}={}&'.format(option, str(value))
 
-def test_last_incomes():
-    """
-    Incomes related test
-    """
-    pass
 
+class TestEmployees(TestCase):
 
-def test_sum():
-    """
-    Incomes related test
-    """
-    pass
+    @pytest.mark.django_db
+    def test_fire(self):
+        # given
+        employee = EmployeeFactory()
+        expected_hired = False
 
+        # when
+        CLIENT.get(_url_detail(EMPLOYEES, employee.pesel, 'fire'))
```

Comment:
As above the `client` can be injected by pytest as there's already a fixture for it:
```
def test_fire(self, client)
```

Reply:
done